pkg/pgmodel/pgx.go

```diff
@@ -904,10 +904,10 @@ func (q *pgxQuerier) Query(query *prompb.Query) ([]*prompb.TimeSeries, error) {
 	return results, nil
 }
 
-func (q *pgxQuerier) getResultRows(startTimestamp int64, endTimestamp int64, matchers []*labels.Matcher) ([]pgx.Rows, error) {
+func (q *pgxQuerier) getResultRows(startTimestamp int64, endTimestamp int64, hints *storage.SelectHints, path []parser.Node, matchers []*labels.Matcher) ([]pgx.Rows, parser.Node, error) {
 	metric, cases, values, err := buildSubQueries(matchers)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
```

Comment:
I wonder if it would be better to start using named-`return` so we can just assign the return values and use `return` without parameters; it might make these diffs cleaner

Reply:
I don't think for this PR